bigtable/client/data.h

```diff
@@ -45,7 +45,7 @@ std::shared_ptr<ClientInterface> CreateDefaultClient(std::string project_id,
                                                      ClientOptions options);
 
 inline std::string CreateInstanceName(std::shared_ptr<ClientInterface> client) {
```

Comment:
Why isn't this called GetInstanceName (and the other one GetTableName)?

I generally think of "Create" as an operation that involves changing some state, so was initially confused.

Reply:
Fixed.

It creates a new object (the string).  But never mind, if it is confusing it should be changed.  Also planning to refactor these things in #144.